### PR TITLE
10 merge overloads

### PR DESCRIPTION
**Pull request by digitaldiina**

### CODE REVIEW

File: docs/api-reference/classes/NSContactAgent/GetAddressByCountry.md [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
- - **contactId** The contact id
- - **countryId** 
+Gets the contact's localized address (LocalizedField[][]).
 
-**Returns:** The address as LocalizedField[][].
+`Address GetAddressByCountry(Integer contactId, Integer countryId)`
+
+## Parameters
+
+| contactId | Integer | The contact ID |
```
Comment (SuperOfficeDevNet) [inline docs/api-reference/classes/NSContactAgent/GetAddressByCountry.md:18]: Add parameter comment: The country ID.

File: docs/api-reference/classes/NSVersionInfo/SetCheckedInDate.md [hunk 2]
```diff
@@ -1,19 +1,28 @@
...
 
-**Parameter:** 
- - **checkedInDate** DateTime
+`SetCheckedInDate(DateTime checkedInDate)`
+
+## Parameters
+
+| Parameter | Type | Description |
+|---|---|---|
+| checkedInDate | DateTime | |
```
Comment (SuperOfficeDevNet) [inline docs/api-reference/classes/NSVersionInfo/SetCheckedInDate.md:20]: Description: The date that this version was checked in and became official.

File: docs/api-reference/classes/SearchEngine/buildSql.md [hunk 3]
```diff
@@ -1,15 +1,19 @@
...
 
+> [!NOTE]
+> For versions 7.0 and newer, the statement returned will be an estimate of the SQL, since the query is sent to NetServer instead.
 
-Note that when running version versions from 7.0, the statement returned will be an estimate of the SQL, since the query is sent to NetServer instead.
-
+## Methods
 
+* SearchEngine.buildSql()
+* SearchEngine.buildSql(String s)
```
Comment (SuperOfficeDevNet) [inline docs/api-reference/classes/SearchEngine/buildSql.md:19]: String parameter description: Determines type of query to build. 
Parameter options:
"select"
"insert"
"update"
"delete"

File: docs/api-reference/classes/StatLib/execute.md [hunk 4]
```diff
@@ -1,14 +1,17 @@
...
+# execute()
 
+Runs the query and aggregation and stores the results. You can choose between storing in a Parser or a StatResult by passing the appropriate object.
 
-* **p:** The parser to store the results in.
-
+## Methods
 
+* StatLib.execute(Parser p)
+* StatLib.execute(StatResult sr)
```
Comment (SuperOfficeDevNet) [inline docs/api-reference/classes/StatLib/execute.md:17]: Parameter Descriptions:
Parser: The Parser instance to store the results in.
StatResult: The StatResult instance to store the results in.

File: docs/api-reference/global-functions/Date.md [hunk 5]
```diff
@@ -1,17 +1,23 @@
...
-description: Date Date(Date p0)
+uid: crmscript_global_date
+title: Date
+description: CRMScript Date constructor
 intellisense: Void.Date
 langref: 1
-keywords: Date(Date)
+keywords: Date(Date), Date(String)
 so.topic: reference
 ---
```
Comment (SuperOfficeDevNet) [inline docs/api-reference/global-functions/Date.md:10]: This overall description does not seem to follow the same conventions established in previous docs.

Append to description: The default value is now.

Perhaps link to, or include examples from [Data types documentation](https://github.com/SuperOfficeDocs/crmscript/blob/main/docs/datatypes/date-type.md).

File: docs/api-reference/global-functions/Float.md [hunk 6]
```diff
@@ -1,17 +1,27 @@
...
-description: Float Float(Float value)
+uid: crmscript_global_float
+title: Float
+description: CRMScript Float constructor
 intellisense: Void.Float
 langref: 1
-keywords: Float(Float)
+keywords: Float(Float), Float(String), Float(Integer)
 so.topic: reference
 ---
```
Comment (SuperOfficeDevNet) [inline docs/api-reference/global-functions/Float.md:10]: This overall description does not seem to follow the same conventions established in other crmscript classes docs.

Perhaps link to, or include examples from [Float types documentation](https://github.com/SuperOfficeDocs/crmscript/blob/main/docs/datatypes/float-type.md).

File: docs/api-reference/global-functions/Integer.md [hunk 7]
```diff
@@ -1,17 +1,23 @@
...
-description: Integer Integer(Integer value)
+uid: crmscript_global_integer
+title: Integer
+description: CRMScript Integer constructor
 intellisense: Void.Integer
 langref: 1
-keywords: Integer(Integer)
+keywords: Integer(Integer), Integer(String value)
 so.topic: reference
 ---
```
Comment (SuperOfficeDevNet) [inline docs/api-reference/global-functions/Integer.md:10]: Same comment as Float and Date..
Consider adding link to or examples from [Integer data type docs](https://github.com/SuperOfficeDocs/crmscript/blob/main/docs/datatypes/integer-type.md)

File: docs/api-reference/global-functions/String.md [hunk 8]
```diff
@@ -1,19 +1,30 @@
...
-description: String String(Byte[] byteArray)
+uid: crmscript_global_string
+title: String
+description: CRMScript String constructor
 intellisense: Void.String
 langref: 1
-keywords: String(Byte[])
+keywords: String(String), String(Byte[]), String(NSStream), String(Byte[],String), String(NSStream,String)
 so.topic: reference
 ---
```
Comment (SuperOfficeDevNet) [inline docs/api-reference/global-functions/String.md:10]: Considering linking to, or include examples from [data types documentation](https://github.com/SuperOfficeDocs/crmscript/blob/main/docs/datatypes/string-type.md).

File: docs/api-reference/global-functions/Time.md [hunk 9]
```diff
@@ -1,22 +1,23 @@
...
-description: Time Time(String time)
+uid: crmscript_global_time
+title: Time
+description: CRMScript Time constructor
 intellisense: Void.Time
 langref: 1
-keywords: Time(String)
+keywords: Time(Time), Time(String)
 so.topic: reference
 ---
```
Comment (SuperOfficeDevNet) [inline docs/api-reference/global-functions/Time.md:10]: Doesn't conform to the standard layout (See [String](https://github.com/SuperOfficeDocs/crmscript/pull/11/files#diff-c72874fdabbc14e3c4e25b587a6d7f229a97f1acce17676d630f97e23cbe6646)).
Considering linking to, or include examples from [data types documentation](https://github.com/SuperOfficeDocs/crmscript/blob/main/docs/datatypes/time-type.md).

File: docs/api-reference/global-functions/XMLNode.md [hunk 10]
```diff
@@ -1,17 +1,23 @@
...
-description: XMLNode XMLNode(String p0)
+uid: crmscript_global_xmlnode
+title: XMLNode
+description: CRMScript XMLNode constructor
 intellisense: Void.XMLNode
 langref: 1
-keywords: XMLNode(String)
+keywords: XMLNode(XMLNode), XMLNode(String)
 so.topic: reference
 ---
```
Comment (SuperOfficeDevNet) [inline docs/api-reference/global-functions/XMLNode.md:10]: Doesn't conform to the standard layout (See String).

**Review by SuperOfficeDevNet — CHANGES_REQUESTED** [review]
Great effort! Just asking for a few changes to level-up to the standard for a few of the files. Also, __every__ parameter must have a description - even if the parameter name seems obvious.

File: docs/api-reference/classes/NSDocumentTemplate/SetDirection.md [hunk 11]
```diff
@@ -1,23 +1,35 @@
...
- - **direction** DocTmplDirection
-     - Enum: 0 = Unknown 
-     - Enum: 1 = Incoming 
-     - Enum: 2 = Outgoing 
-     - Enum: 3 = SaintAll 
+## Parameters
+
+| Parameter | Type | Description |
+|---|---|---|
+| direction | DocTmplDirection | |
```
[the remark this answers is not in the document]
Comment (digitaldiina) [reply docs/api-reference/classes/NSDocumentTemplate/SetDirection.md:16]: Thanks for the link

**Comment by digitaldiina** [comment]
Pushed the requested changes. OK now?

File: docs/api-reference/classes/NSVersionInfo/SetCheckedInDate.md [hunk 12]
```diff
@@ -1,19 +1,28 @@
 ﻿---
-uid: crmscript_ref_NSVersionInfo_SetCheckedInDate
-title: SetCheckedInDate(DateTime checkedInDate)
+uid: crmscript_class_nsversioninfo_setcheckedindate
+title: SetCheckedInDate()
+description: CRMScript method in the NSVersionInfo class that sets the date that this version was checked in
 intellisense: NSVersionInfo.SetCheckedInDate
-keywords: NSVersionInfo, GetCheckedInDate
+keywords: NSVersionInfo, GetCheckedInDate SetCheckedInDate(DateTime)
```
Comment (SuperOfficeDevNet) [inline docs/api-reference/classes/NSVersionInfo/SetCheckedInDate.md:6]: I think there should be a comma between keywords

**Review by SuperOfficeDevNet — CHANGES_REQUESTED** [review]
Just one more correction.

File: docs/api-reference/classes/NSVersionInfo/SetCheckedInDate.md (hunk 12 above)
Comment (digitaldiina) [reply docs/api-reference/classes/NSVersionInfo/SetCheckedInDate.md]: done

File: docs/api-reference/classes/NSVersionInfo/SetCheckedInDate.md [hunk 13]
```diff
@@ -1,19 +1,28 @@
 ﻿---
-uid: crmscript_ref_NSVersionInfo_SetCheckedInDate
-title: SetCheckedInDate(DateTime checkedInDate)
+uid: crmscript_class_nsversioninfo_setcheckedindate
+title: SetCheckedInDate()
+description: CRMScript method in the NSVersionInfo class that sets the date that this version was checked in
 intellisense: NSVersionInfo.SetCheckedInDate
-keywords: NSVersionInfo, GetCheckedInDate
+keywords: NSVersionInfo, GetCheckedInDate, SetCheckedInDate(DateTime)
```
Comment (SuperOfficeDevNet) [inline docs/api-reference/classes/NSVersionInfo/SetCheckedInDate.md:6]: Awesome sauce!

**Review by SuperOfficeDevNet — APPROVED** [review]
Everything looks great now, thanks for this submittion!

**Pull request merged**